src/main/java/frc/robot/RobotContainer.java

```diff
@@ -44,6 +55,10 @@
      */
     public RobotContainer() {
         driveSubsystem.setDefaultCommand(driveFromController);
+        shooterCommand.addRequirements(shooterSubsystem);
+        intakeCommand.addRequirements(intakeSubsystem);

```

Comment:
```suggestion
```

I think requirements are already added in the `ShooterCommand` and `IntakeCommand` constructors.

Reply:
Lines deleted, per request.